google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -231,6 +230,34 @@ private LogEntry logEntryFor(ILoggingEvent e) {
     return builder.build();
   }
 
+  @InternalApi
+  static void writeStack(IThrowableProxy throwProxy, String prefix, StringBuilder payload) {
+    if (throwProxy == null) {
+      return;
+    }
+    payload
+        .append(prefix)
+        .append(throwProxy.getClassName())
+        .append(": ")
+        .append(throwProxy.getMessage())
+        .append('\n');
+    StackTraceElementProxy[] trace = throwProxy.getStackTraceElementProxyArray();
```

Comment:
Just as a sanity check, do we know that this is a stable API that we are using? (Or maybe it doesn't matter since google-cloud-logging-logback is alpha...)

Reply:
I believe it is stable. We depend on version `1.2.3`.